app/src/main/java/atlant/moviesapp/model/Account.java

```diff
@@ -158,6 +158,75 @@ public void setWatchListSeries(List<Integer> watchListSeries) {
         this.watchListSeries = watchListSeries;
     }
 
+    public void addFavouriteMovie(Integer id) {
+        this.favouriteMovies.add(id);
+    }
+
+    public void addFavouriteShow(Integer id) {
+        this.favouriteSeries.add(id);
+    }
+
+    public void addWatchlistMovie(Integer id) {
+        this.watchListMovies.add(id);
+    }
+
+    public void addWatchlistShow(Integer id) {
+        this.watchListSeries.add(id);
+    }
+
+    public void addMovieRating(Integer id) {
+        this.ratedMovies.add(id);
+    }
+
+    public void addShowRating(Integer id) {
+        this.ratedSeries.add(id);
+    }
+
+    public void removeFavouriteMovie(Integer id) {
+        for (int i = 0; i < this.getFavouriteMovies().size(); i++) {
+            if (this.getFavouriteMovies().get(0).equals(id))
```

Comment:
Are you sure, you want to check is the id (sent as the parameter) equal to the the first (at the 0 position) element of the list,  for list's size number of times ??? Really ? Please check the 6 methods that follow this one
!Important